server/src/db/mongo/portfolio.ts

```diff
@@ -0,0 +1,57 @@
+import {getDB, errorCodes} from './db'
+
+export async function init(): Promise<void> {
+  try {
+    await getDB().createCollection('portfolio')
+  } catch (err) {
+    if (err.code === errorCodes.namespaceExists) {
+      return
+    }
+
+    console.error('Unexpected error initializing portfolio collection', err)
+    throw new Error('FailedInitPortfolio')
+  }
+
+  try{
+    await getDB().collection('portfolio').createIndex({businessId: 1})
+  } catch(err) {
+    console.error('Unexpected error creating businessId index', err)
+    throw new Error('FailedInitPortfolio')
+  }
+}
+
+export async function insert(
+  description: string,
+  file: string,
+  businessId: string,
+): Promise<void> {
+  try{
+    const result = await getDB().collection("portfolio").insertOne({
+      description, file, businessId
+    })
+    return result.insertedId.toHexString()
+  } catch (err){
+    console.error('Unexpected error inserting portfolio', err)
+    throw new Error('FailedInsertPortfolioItem')
+  }
+}
+
+interface Portfolio {
+  id: string
+  file: string
+  description: string
+  businessID: string
+}
+
+export async function get(businessId: string): Promise<Array<Portfolio>> {
+  let result : Portfolio[] = [];
+  try{
+    result = await getDB().collection("portfolio").find(
+      { businessId: businessId}
+    ).toArray();
```

Comment:
I realized that we probably do want to have the id on here as a string just like we did in the PostgreSQL version. This should do the trick:
```suggestion
    ).toArray().map(item => ({
      id: item._id.toHexString(),
      file: item.file,
      description: item.description,
      businessId: item.businessId,
    }));
```

Reply:
Then you can add the id back into your tests if you want.

Reply:
Done!